ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+[//]: # (Note: See https://github.com/HabitRPG/habitrpg/issues/2760 for more info)
```

Comment:
Add something like this:

Before logging this issue, look through common problems at https://github.com/HabitRPG/habitrpg/issues?q=label%3AMemorable
If you find your issue there, read at least the first post to see if there is a workaround for you.